ELIFECYCLE error

try running either of bump scripts like `npm run bump-major-beta` results in:

> @axa-ch/patterns-library@2.0.0-beta prebump-major-beta /Users/axawinterthur/dev/axa-ch_style-html-guide
 test
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @axa-ch/patterns-library@2.0.0-beta prebump-major-beta: `test`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @axa-ch/patterns-library@2.0.0-beta prebump-major-beta script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/axawinterthur/.npm/_logs/2017-12-13T16_43_48_507Z-debug.log

--- a/package.json
+++ b/package.json
@@ -23,10 +23,10 @@
     "build-icons-prod": "node stack/tasks/icons.js PROD",
     "build-dev": "node stack/tasks/cleanup.js DEV && npm run build-css-dev && node stack/tasks/build.html.js DEV && node stack/tasks/build-examples.js DEV && npm run build-js-dev && npm run build-icons-dev",
     "build": "node stack/tasks/cleanup.js PROD && node stack/tasks/build.html.js PROD && node stack/tasks/build-examples.js PROD && npm run build-css-prod && npm run build-js-prod && npm run build-icons-prod",
-    "bump-major": "test && npmversion --increment major",
-    "bump-minor": "test && npmversion --increment minor",
-    "bump-patch": "test && npmversion --increment patch",
-    "bump-major-beta": "test && npmversion --increment major --preid beta"
+    "bump-major": "npm run test && npmversion --increment major",
+    "bump-minor": "npm run test && npmversion --increment minor",
+    "bump-patch": "npm run test && npmversion --increment patch",
+    "bump-major-beta": "npm run test && npmversion --increment major --preid beta"
   },
   "watch": {
     "build-dev": {

--- a/package.json
+++ b/package.json
@@ -26,7 +26,7 @@
     "bump-major": "npm run test && npmversion --increment major",
     "bump-minor": "npm run test && npmversion --increment minor",
     "bump-patch": "npm run test && npmversion --increment patch",
-    "bump-major-beta": "npm run test && npmversion --increment major --preid beta"
+    "bump-major-beta": "npm run test && npmversion --increment major --preid beta --force-preid"
   },
   "watch": {
     "build-dev": {

